requirements.txt

```diff
@@ -9,3 +9,7 @@ pypandoc
 pytest-benchmark
 pytest-docker-pexpect
 twine
+psutil
+decorator
+pyte
+
```

Comment:
I believe it's common practice/courtesy to include all required dependencies in requirements.txt so that way users don't have to dig through files to determine what dependency they are missing? Is there a good reason to only include a partial list of dependencies in requirements.txt?

Reply:
The reason is that it's a Python package. User requirements — those required for the tool to work — are defined in `setup.py`. They're the packages that are installed along with `pip install thefuck`. That list also serves as a guide to dependency definition in other package managers. Take the [Homebrew](https://github.com/Homebrew/homebrew-core/blob/6de8c1455224f0c7949aa103a6cf5721a7dcdde5/Formula/thefuck.rb#L22-L50), for example, or [APT](https://github.com/scorphus/thefuck-debian/blob/557ab37645a53e514b9afd7d4f44941be717d447/debian/control#L24-L31).

`requirements.txt` contains development dependencies — these are required to run tests, for instance. [CONTRIBUTING.md](https://github.com/nvbn/thefuck/blob/master/CONTRIBUTING.md#developing) has directives to setup a development environment.

P.S.: Development — or test — dependencies can also be declared in `setup.py`: [example](https://github.com/scorphus/walt/blob/2205d0ac3f9869dc93002a0d12e7e86e0b90e259/setup.py#L68). In such cases, [setup directives](https://github.com/scorphus/walt/blob/2205d0ac3f9869dc93002a0d12e7e86e0b90e259/Makefile#L17) are a bit different too.

Reply:
Thanks for the details! Will make the adjustments